tools/mason/MasonNew.chpl

```diff
@@ -91,13 +103,112 @@ proc masonNew(args) throws {
   }
 }
 
+/*
+  Starts an interactive session to create a
+  new library project.
+*/
+proc beginInteractiveSession(isInit: bool, defaultPackageName: string) throws {
+  writeln("This is an interactive session to walk you through creating a library ",
+   "project using Mason.\nThe following queries covers the common items required to ",
+   "create the project.\nSuggestions for defaults are also provided which will be ",
+   "considered if no input is given.\n");
+  writeln("Press ^C to quit interactive mode.");
+  var packageName: string;
+  var version: string;
+  var chapelVersion: string;
+  var gotCorrectPackageName = false;
+  var gotCorrectPackageVersion = false;
+  var gotCorrectChapelVersion = false;
+  while(1){
+    try {
+      if gotCorrectPackageName == false {
+        write("Package name ");
+        if isInit == true then write("(" + defaultPackageName + ") ");
+        write(": ");
+        IO.stdout.flush();
```

Comment:
No change needed, but note:

When using a package (`use IO;`), we do not require qualified access to module identifiers, so you could also just write `stdout.flush();`.

When importing a package (`import IO;`), you must qualify module identifiers like you have done here. This is new as of Chapel 1.21.0.

I personally prefer qualified accesses most of the time, so I like what you've done here - just making sure you are aware.